spec/requests/api/v2/providers/courses/update_sites_spec.rb

```diff
@@ -10,7 +10,9 @@
     ActionController::HttpAuthentication::Token.encode_credentials(token)
   end
 
-  let(:course) { create :course, provider: provider }
+  let(:course) { create :course, provider: provider, site_statuses: [site_status], subjects: [subject] }
+  let(:site_status) { build(:site_status) }
+  let(:subject) { build(:subject, subject_name: 'primary') }
```

Comment:
On the back of the other comments, just noticed this use of `subject` here. Could we use `primary_subject` here?

Reply:
Done. Thanks for the review Misha.